addons/l10n_ad/models/account_tax.py

```diff
@@ -0,0 +1,22 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountTax(models.Model):
+    _inherit = "account.tax"
+
+    l10n_ad_exempt_reason = fields.Selection(
+        selection=[
+            ("E1", "No exempt"),
```

Comment:
And is it normal that you have a Selection with only one possibility? Shouldn't it be a Boolean then?